hostapd/init.sls

```diff
@@ -23,7 +28,7 @@ hostapd_activate:
   file.replace:
     - name: {{ map.defaults_file }}
     - pattern: "^(|#)DAEMON_CONF=.*$"
-    - repl: "DAEMON_CONF='{% for card in salt['pillar.get']('hostapd:cardlist', {}).keys() %}{{ card2conf(card, map) }} {% endfor %}'"
+    - repl: "DAEMON_CONF='{{ daemon_conf.values()|join(" ") }}'"
```

Comment:
I doubt if most outer double quotes are necessary and would not make any interference?

Reply:
@vutny 
I removed the double quotes. Hopefully nobody uses colons in his filenames. :-)